argus/backend/service/argus_service.py

```diff
@@ -398,11 +423,52 @@ def update_schedule_comment(self, payload: dict) -> dict:
             "newComment": new_comment,
         }
 
+    def update_schedule(self, release_id: UUID | str, schedule_id: UUID | str, old_tests: list[UUID | str], new_tests: list[UUID | str], comments: dict[str, str], assignee: UUID | str):
```

Comment:
In case of 2 concurrent users updating the same schedule old_tests might be outdated and cause errors later in adding tests. I suppose

Reply:
the resulting set in this case will be whatever was removed from old tests which is fine, however `new_tests` could cause duplication, in which case it might be a good idea to first check if we are not creating a second test entity. This isn't that big of a deal, as it will just be removed when schedule is deleted and it doesn't affect logic of assigning.